docs/core_docs/docs/integrations/chat/yandex.mdx

```diff
@@ -19,6 +19,16 @@ Next, you have two authentication options:
 
 ## Usage
 
+You'll first need to install the `@langchain/yandex` package:
+
+:::tip
```

Comment:
FYI just added a component for this:

https://github.com/langchain-ai/langchainjs/pull/3859/files#diff-4abc024fbd91edd4684e6c72069d70fa8dc66d051ee21ce675109344b10c218eR63